packages/devtools/src/index.js

```diff
@@ -30,16 +32,23 @@ export default class DevTools {
         const pages = await browser.pages()
         const driver = new DevToolsDriver(browser, pages)
         const sessionId = uuidv4()
-        const [browserName, browserVersion] = (await browser.version()).split('/')
+        const userAgent = uaParserJs(await browser.userAgent())
```

Comment:
Not the coding ninja like you, but can we also deconstruct the needed data from the userAgent making it more readable when we use the values like  `userAgent.browser.name` and `userAgent.browser.version`

Reply:
yeah .. wanted to do `const { browser } = ...` but saw that `browser` is already defined so I kept it this way.